### DISCUSSION TITLE
Endpoint badge example with logoSvg

### DISCUSSION THREAD

**Discussion by djeanner** [badges/shields#6716] [Q&A] [answered]
I miss badly and example of custom badge with a svg (using logoSvg:)

**Comment by calebcartwright** [comment]
Does this help?

https://github.com/badges/shields/blob/master/doc/logos.md#custom-logos

**Comment by chris48s** [comment, marked as answer]
`logoSvg:` sounds like we're talking about the [endpoint badge](https://shields.io/endpoint)
Maybe this is what you're after?
https://raw.githubusercontent.com/chris48s/shields-endpoint-test/2958b1745abdc566152d54d336550a29c1ccb553/badge1.json
https://img.shields.io/endpoint.svg?url=https%3A//raw.githubusercontent.com/chris48s/shields-endpoint-test/2958b1745abdc566152d54d336550a29c1ccb553/badge1.json

**Reply by djeanner** [reply]
yes, this is it! Thank you!

**Reply by pabloagn** [reply]
You can use the following resource to test:
https://www.svgviewer.dev/
